### DISCUSSION TITLE
An Error happened when I using the pyg=2.3.0

### DISCUSSION THREAD

**Discussion by StefanIsSmart** [pyg-team/pytorch_geometric#8959] [General]
I will update later

**Comment by rusty1s** [comment]
How can I run your example?

**Reply by StefanIsSmart** [reply]
Thank you again!
I will rewrite the debug report again, and give you a mini processed data sample. Do you think it's okay?

**Reply by StefanIsSmart** [reply]
Hello, when I want to reproduce the error and give you a mini-sample,
I find the result is the processed data is from the PyG2.2.0, 
If I regenerate the processed data, the error is gone!

**Reply by StefanIsSmart** [reply]
Hi @rusty1s 

I find that:

The processed data from PyG2.5.0 and run with PyG2.2.0 will happened:
```
  File "/export/disk3/why/software/Miniforge3/envs/PyG250/lib/python3.8/site-packages/torch_geometric/data/in_memory_dataset.py", line 319, in __getattr__
    raise AttributeError(f"'{self.__class__.__name__}' object has no "
AttributeError: 'MyDataset_batch_ShortQM9' object has no attribute '_data'
```

The processed data from PyG2.2.0 and run with PyG2.5.0 / PyG2.3.0 will happened:
```
  File "/export/disk3/why/software/Miniforge3/envs/PyG220/lib/python3.7/site-packages/torch_geometric/data/in_memory_dataset.py", line 83, in get
    cls=self.data.__class__,
AttributeError: 'MyDataset_batch_ShortQM9' object has no attribute 'data'
```

**Reply by rusty1s** [reply]
I cannot reproduce this on any dataset in PyG (e.g, tested with `TUDataset`), so this seems to be a special issue. Still curious how I can run your example to help you :(

**Comment by StefanIsSmart** [comment]
> How can I run your example?



> How can I run your example?



> How can I run your example?
